src/pipeline.py

```diff
@@ -110,9 +131,10 @@ def run_recur(source_executor, item, dataUpdater):
         metadata = iExecutor.get_metadata(next_item)
         if metadata.is_cancelled == True:
             metadata.add_tag('state', 'processed')
-            dataUpdater.safe_run(metadata)
+            # dataUpdater.safe_run(metadata)
```

Comment:
Why did you comment these out?

Reply:
oh ya thats so I dont accidentally break the database while testing